packages/location-field/src/utils.ts

```diff
@@ -64,3 +70,31 @@ export const getCombinedLabel = (properties: Properties): string => {
   }
   return properties?.label || "";
 };
+
+/**
+ * Helper method to append text in parentheses to some other text,
+ * if the added text is not null or blank.
+ */
+export const addInParentheses = (
+  mainText: string,
+  extraText?: string
+): string => {
+  return extraText && extraText !== ""
+    ? `${mainText} (${extraText})`
+    : mainText;
+};
+
+/**
+ * Helper function to assemble a geocoder error message.
+ */
+export const getGeocoderErrorMessage = (
+  intl: IntlShape,
+  errorText?: string
+): string => {
+  const geocoderUnreachableText = intl.formatMessage({
+    description: "Geocoder unreachable status",
+    id: "otpUi.LocationField.geocoderUnreachable"
+  });
+
+  return addInParentheses(geocoderUnreachableText, errorText);
```

Comment:
Do we think seeing the `errorText` is helpful to users? This error message feels a little technical: 
![image](https://user-images.githubusercontent.com/115499534/232891633-8cecd5ff-7a43-47e2-b988-d8774bff864f.png)

Reply:
This was the existing behavior, but good question, @miles-grant-ibigroup what do you think?